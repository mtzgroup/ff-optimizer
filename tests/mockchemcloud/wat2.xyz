3
converted from 3.pdb
O	-0.610	1.067	0.460
H	-0.316	0.176	0.353
H	-0.920	1.965	0.630
