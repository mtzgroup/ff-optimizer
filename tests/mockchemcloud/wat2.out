| Startfile from command line: wat2.in


|       ***********************************************************
|       *                 TeraChem v1.9-2025.01-dev               *
|       *                   Development Version                   *
|       *           Chemistry at the Speed of Graphics!           *
|       ***********************************************************
|       * This program may only be used in connection with        *
|       * a valid license from PetaChem, LLC. Use of this program *
|       * or results thereof indicates acceptance of all terms    *
|       * and conditions stated in the license and that a valid   *
|       * license agreement between the user and PetaChem, LLC    *
|       * exists. PetaChem, LLC does not warrant the correctness  *
|       * of results or their suitability for any purpose.        *
|       * Please email bugs, suggestions, and comments to         *
|       *                  help@petachem.com                      *
|       *                                                         *
|       ***********************************************************


|       ***********************************************************
|       *  Compiled by root         Wed Feb  5 07:16:34 PM PST 2025  *
|       *  Supported architecture SMs: 52 61 70 80 90             *
|       *  Build cuda_12.2.r12.2/compiler.33053471_0              *
|       *  Git Version:                               *
|       ***********************************************************


| Job started   Wed Feb 19 10:44:23 2025
| On fire-09-07 (available memory: 50750 MB)

| ######################################### RUNTIME INFO ##########################################
| terachem  wat2.in 

|  NVRM version: NVIDIA UNIX x86_64 Kernel Module  525.89.02  Wed Feb  1 23:23:25 UTC 2023
|  GCC version:  gcc version 11.4.0 (Ubuntu 11.4.0-1ubuntu1~22.04) 

|  	linux-vdso.so.1 (0x00007ffc6e9be000)
|  	libcurl.so.4 => /lib/x86_64-linux-gnu/libcurl.so.4 (0x00007f27e3deb000)
|  	libz.so.1 => /lib/x86_64-linux-gnu/libz.so.1 (0x00007f27e3dcf000)
|  	libimf.so => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libimf.so (0x00007f27e39e5000)
|  	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f27e38fc000)
|  	libmpi.so.12 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libmpi.so.12 (0x00007f27e1e00000)
|  	libdftbplus.so => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libdftbplus.so (0x00007f27e1aa8000)
|  	libfsockets.so => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libfsockets.so (0x00007f27e38f6000)
|  	libxmlf90.so => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libxmlf90.so (0x00007f27e1a7d000)
|  	libthcbox.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libthcbox.so.1 (0x00007f27dfc60000)
|  	libOpenMM.so => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libOpenMM.so (0x00007f27df806000)
|  	libFlexiBLE.so => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libFlexiBLE.so (0x00007f27e38e1000)
|  	libnvToolsExt.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libnvToolsExt.so.1 (0x00007f27df400000)
|  	libgpubox.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libgpubox.so.1 (0x00007f27e38d6000)
|  	libcpubox.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libcpubox.so.1 (0x00007f27df800000)
|  	libtensorbox.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libtensorbox.so.1 (0x00007f27df7e4000)
|  	libiobox.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libiobox.so.1 (0x00007f27df7de000)
|  	libintbox.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libintbox.so.1 (0x00007f27798a1000)
|  	libatmx2cbox.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libatmx2cbox.so.1 (0x00007f27df71d000)
|  	libperiodicbox.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libperiodicbox.so.1 (0x00007f275ea1a000)
|  	libsqmbox.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libsqmbox.so.1 (0x00007f275e66e000)
|  	libgridbox.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libgridbox.so.1 (0x00007f275c2a4000)
|  	libdftbox.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libdftbox.so.1 (0x00007f27df61d000)
|  	libcibox.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libcibox.so.1 (0x00007f275bc55000)
|  	libccbox.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libccbox.so.1 (0x00007f275b7c4000)
|  	libsimbox.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libsimbox.so.1 (0x00007f275b79c000)
|  	liboptbox.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/liboptbox.so.1 (0x00007f275b767000)
|  	libprotobuf.so.25 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libprotobuf.so.25 (0x00007f275b494000)
|  	libcommbox.so.1 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libcommbox.so.1 (0x00007f275b433000)
|  	libssl.so.3 => /lib/x86_64-linux-gnu/libssl.so.3 (0x00007f275b38f000)
|  	libcrypto.so.3 => /lib/x86_64-linux-gnu/libcrypto.so.3 (0x00007f275af4c000)
|  	libcuda.so.1 => /lib/x86_64-linux-gnu/libcuda.so.1 (0x00007f275926c000)
|  	libcudart.so.12 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libcudart.so.12 (0x00007f2758e00000)
|  	libcublas.so.12 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libcublas.so.12 (0x00007f2752600000)
|  	libcufft.so.11 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libcufft.so.11 (0x00007f2747800000)
|  	libcusparse.so.12 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libcusparse.so.12 (0x00007f2737a00000)
|  	libcusolver.so.11 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libcusolver.so.11 (0x00007f2730800000)
|  	libcublasLt.so.12 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libcublasLt.so.12 (0x00007f270e000000)
|  	libsvml.so => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libsvml.so (0x00007f270c9d1000)
|  	libirng.so => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libirng.so (0x00007f270c600000)
|  	libstdc++.so.6 => /lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007f270c3d4000)
|  	libiomp5.so => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libiomp5.so (0x00007f270be00000)
|  	libcilkrts.so.5 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libcilkrts.so.5 (0x00007f270ba00000)
|  	libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f275924c000)
|  	libintlc.so.5 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libintlc.so.5 (0x00007f27591d4000)
|  	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f270b7d7000)
|  	/lib64/ld-linux-x86-64.so.2 (0x00007f27e3ea1000)
|  	libnghttp2.so.14 => /lib/x86_64-linux-gnu/libnghttp2.so.14 (0x00007f27591aa000)
|  	libidn2.so.0 => /lib/x86_64-linux-gnu/libidn2.so.0 (0x00007f2759189000)
|  	librtmp.so.1 => /lib/x86_64-linux-gnu/librtmp.so.1 (0x00007f275916a000)
|  	libssh.so.4 => /lib/x86_64-linux-gnu/libssh.so.4 (0x00007f27590fd000)
|  	libpsl.so.5 => /lib/x86_64-linux-gnu/libpsl.so.5 (0x00007f27590e9000)
|  	libgssapi_krb5.so.2 => /lib/x86_64-linux-gnu/libgssapi_krb5.so.2 (0x00007f27525ac000)
|  	libldap-2.5.so.0 => /lib/x86_64-linux-gnu/libldap-2.5.so.0 (0x00007f275254d000)
|  	liblber-2.5.so.0 => /lib/x86_64-linux-gnu/liblber-2.5.so.0 (0x00007f27590d8000)
|  	libzstd.so.1 => /lib/x86_64-linux-gnu/libzstd.so.1 (0x00007f2737931000)
|  	libbrotlidec.so.1 => /lib/x86_64-linux-gnu/libbrotlidec.so.1 (0x00007f27590ca000)
|  	librt.so.1 => /lib/x86_64-linux-gnu/librt.so.1 (0x00007f27df60c000)
|  	libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007f27590c5000)
|  	libifport.so.5 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libifport.so.5 (0x00007f2758dd6000)
|  	libifcoremt.so.5 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libifcoremt.so.5 (0x00007f270c25a000)
|  	libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f27590c0000)
|  	libnvJitLink.so.12 => /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/lib/libnvJitLink.so.12 (0x00007f2708600000)
|  	libunistring.so.2 => /lib/x86_64-linux-gnu/libunistring.so.2 (0x00007f270bc56000)
|  	libgnutls.so.30 => /lib/x86_64-linux-gnu/libgnutls.so.30 (0x00007f2708415000)
|  	libhogweed.so.6 => /lib/x86_64-linux-gnu/libhogweed.so.6 (0x00007f27477b8000)
|  	libnettle.so.8 => /lib/x86_64-linux-gnu/libnettle.so.8 (0x00007f27378eb000)
|  	libgmp.so.10 => /lib/x86_64-linux-gnu/libgmp.so.10 (0x00007f2737869000)
|  	libkrb5.so.3 => /lib/x86_64-linux-gnu/libkrb5.so.3 (0x00007f2730735000)
|  	libk5crypto.so.3 => /lib/x86_64-linux-gnu/libk5crypto.so.3 (0x00007f270c9a2000)
|  	libcom_err.so.2 => /lib/x86_64-linux-gnu/libcom_err.so.2 (0x00007f27590b6000)
|  	libkrb5support.so.0 => /lib/x86_64-linux-gnu/libkrb5support.so.0 (0x00007f27590a8000)
|  	libsasl2.so.2 => /lib/x86_64-linux-gnu/libsasl2.so.2 (0x00007f2752532000)
|  	libbrotlicommon.so.1 => /lib/x86_64-linux-gnu/libbrotlicommon.so.1 (0x00007f2737846000)
|  	libp11-kit.so.0 => /lib/x86_64-linux-gnu/libp11-kit.so.0 (0x00007f27082da000)
|  	libtasn1.so.6 => /lib/x86_64-linux-gnu/libtasn1.so.6 (0x00007f27477a0000)
|  	libkeyutils.so.1 => /lib/x86_64-linux-gnu/libkeyutils.so.1 (0x00007f2758dcd000)
|  	libresolv.so.2 => /lib/x86_64-linux-gnu/libresolv.so.2 (0x00007f270c98e000)
|  	libffi.so.8 => /lib/x86_64-linux-gnu/libffi.so.8 (0x00007f270c981000)
| #################################################################################################

| Checking Amazon Cloud instance ID...   Not found
| Cannot find license.dat file /global/software/TeraChem/2025.01-intel-2023.2.0-CUDA-12.2.1/license.dat


| TRYING THE NETWORK LICENSE...
| Connecting to license server '54.208.252.40' port '8877'...
| Connected!
| Checking your license...

| **************************************************************
|  Greetings, Martinez Group! You have 1000 licenses in total
|  IN USE: 44
|  AVAILABLE: 956
| **************************************************************

-----------------------------------
Processed Input file:
                   method uhf
                   charge 1
                 spinmult 2
              coordinates wat2.xyz
                     resp yes
                      run gradient
                    basis 6-31g*
                  threall 1e-15
                 convthre 1e-6
                precision double
-----------------------------------
Jobname: wat2
Scratch data will be generated
Scratch directory: ./scr.wat2
| Random number seed: 1174095197

XYZ coordinates wat2.xyz
Molden File Output: ./scr.wat2/wat2.molden
Using basis set: 6-31gs
dmrgstart not found
Spin multiplicity: 2
DIIS will use up to 10 vectors.
Condition number limit for the DIIS overlap matrix is            0
WF convergence threshold: 1.00e-06
Using DIIS algorithm to converge WF
Maximum number of SCF iterations: 100
Incremental fock with rebuild every 8 iterations
Will switch to conventional Fock if diffuse functions are detected
X-matrix tolerance: 1.00e-04
PRECISION: DOUBLE
TeraChem will select linear algebra engine
DFT Functional requested: hf
Method: Hartree-Fock 
Wavefunction: UNRESTRICTED
Initial guess generated by maximum overlap

********************************************
**** SINGLE POINT GRADIENT CALCULATIONS ****
********************************************
SCF:   Initial guess is taken from previous step
Starting TensorBox...
| TeraChem will select linear algebra engine
| Using cuSolver for matrix diagonalization

| using 1 out of 1 CUDA devices
|   Device 0: NVIDIA GeForce GTX TITAN X, 12207MB, CC 5.2 -- CPU THREAD 0
| -------------------------------------------------------------------
| CPU Memory Available: 18127.88 MegaWords 
| GPU Memory Available: 1269.88 MegaWords 
| Maximum recommended basis set size: 13300 basis functions
|  (limited by GPU memory)
| -------------------------------------------------------------------
| Initialize GPUBox with 128 MW of memory

****** QM coordinates ******
O        -0.6100000000         1.0670000000         0.4600000000  
H        -0.3160000000         0.1760000000         0.3530000000  
H        -0.9200000000         1.9650000000         0.6300000000  

Basis set:       6-31gs
Total atoms:     3
Total charge:    1
Total electrons: 9 (5-alpha, 4-beta)
Number electrons modeled by ECPs: 0
Total orbitals:  19
Total AO shells: 10 (7 S-shells; 2 P-shells; 1 D-shells; 0 F-shells; 0 G-shells)
Spin multiplicity: 2
Nuclear repulsion energy (QM atoms): 9.146794238524 a.u.


                      *** Start SCF Iterations ***

 Iter     DIIS Error       Energy change         Energy        Time(s)
------------------------------------------------------------------------
| Purifying alpha guess...
| Trace of guess   5.00000000, trace of guess*guess   5.00000000,  electron number   5.00000000 
| Guess is well-conditioned: purify...
| Purification run on CPU (double precision)  using TC2 purification.
| Density matrix purification finished after a total of 0 matrix multiplies
| Purifying beta guess...
| Trace of guess   4.00000000, trace of guess*guess   4.00000000,  electron number   4.00000000 
| Guess is well-conditioned: purify...
| Purification run on CPU (double precision)  using TC2 purification.
| Density matrix purification finished after a total of 0 matrix multiplies
|    1     1.1084794243     -74.7631935225     -74.7631935225     0.09  
|    2     0.3841486222      -0.7020136336     -75.4652071561     0.04  
|    3     0.0554309881      -0.1085176733     -75.5737248295     0.04  
|    4     0.0097464345      -0.0031677851     -75.5768926146     0.04  
|    5     0.0021736002      -0.0001686170     -75.5770612316     0.04  
|    6     0.0006588559      -0.0000105006     -75.5770717322     0.04  
|    7     0.0001562655      -0.0000008756     -75.5770726077     0.04  
|    8     0.0001310587      -0.0000001964     -75.5770728041     0.03  
|    9     0.0001313905      -0.0000000901     -75.5770728942     0.03  
|   10     0.0001314754      -0.0000001148     -75.5770730091     0.03  
|   11     0.0001314666      -0.0000000618     -75.5770730708     0.03  
|   12     0.0001314677      -0.0000000303     -75.5770731011     0.04  
|   13     0.0001314810      -0.0000003106     -75.5770734117     0.03  
|   14     0.0001315121      -0.0000006056     -75.5770740173     0.03  
|   15     0.0001314880      +0.0000005171     -75.5770735002     0.03  
|   16     0.0032307015      +0.0000552020     -75.5770182982     0.03  
|   17     0.0042637787      +0.0000183978     -75.5769999004     0.03  
|   18     0.0034086584      -0.0000155216     -75.5770154220     0.04  
|   19     0.0029141693      -0.0000079854     -75.5770234074     0.03  
|   20     0.0022030164      -0.0000104077     -75.5770338151     0.03  
|   21     0.0022320091      +0.0000004201     -75.5770333951     0.03  
|   22     0.0011290921      -0.0000147857     -75.5770481807     0.03  
|   23     0.0009289418      -0.0000024277     -75.5770506085     0.03  
|   24     0.0007579295      -0.0000016580     -75.5770522665     0.03  
|   25     0.0002155475      -0.0000071986     -75.5770594651     0.04  
|   26     0.0001317873      -0.0000025522     -75.5770620173     0.03  
|   27     0.0001444975      +0.0000157457     -75.5770462716     0.03  
|   28     0.0218231421      +0.0004313316     -75.5766149400     0.03  
|   29     0.0199528549      -0.0000543944     -75.5766693344     0.03  
|   30     0.0200969978      +0.0000049134     -75.5766644210     0.03  
|   31     0.0342721745      +0.0005000543     -75.5761643667     0.03  
|   32     0.0173420819      -0.0005755935     -75.5767399602     0.04  
|   33     0.0112274471      -0.0001375457     -75.5768775059     0.03  
|   34     0.0095284376      -0.0000311375     -75.5769086435     0.03  
|   35     0.0229253753      +0.0003168834     -75.5765917601     0.03  
|   36     0.0223347724      -0.0000183729     -75.5766101330     0.03  
|   37     0.0008597544      -0.0003875765     -75.5769977095     0.03  
|   38     0.0004966379      +0.0000093321     -75.5769883773     0.03  
|   39     0.0165652054      +0.0002273557     -75.5767610216     0.03  
|   40     0.0164185877      -0.0000026876     -75.5767637092     0.03  
|   41     0.0134867245      -0.0000627158     -75.5768264250     0.03  
|   42     0.0136029424      +0.0000021860     -75.5768242390     0.03  
|   43     0.0140521904      +0.0000087573     -75.5768154817     0.04  
|   44     0.0057090766      -0.0001205007     -75.5769359824     0.03  
|   45     0.0029017246      -0.0000103912     -75.5769463736     0.03  
|   46     0.0074567803      +0.0000448906     -75.5769014830     0.03  
|   47     0.0080044452      +0.0000052535     -75.5768962295     0.03  
|   48     0.0036048261      -0.0000352758     -75.5769315053     0.03  
|   49     0.0003626556      -0.0000092469     -75.5769407522     0.03  
|   50     0.0002734889      +0.0000000874     -75.5769406647     0.03  
|   51     0.0002331566      +0.0000000147     -75.5769406501     0.03  
|   52     0.0000113251      -0.0000000365     -75.5769406866     0.03  
|   53     0.0000316020      +0.0000000009     -75.5769406857     0.03  
|   54     0.0000207403      -0.0000000005     -75.5769406862     0.03  
|   55     0.0000000846      -0.0000000004     -75.5769406865     0.03  
------------------------------------------------------------------------
FINAL ENERGY: -75.5769406865 a.u.
| WARNING: Final energy is higher than the lowest energy by   0.00013333.
HOMO-LUMO GAP: alpha: 23.365 eV
               beta:  17.925 eV 
CENTER OF MASS: {-0.610895, 1.067392, 0.463525} ANGS
DIPOLE MOMENT: {-0.045342, 0.053021, 0.132794} (|D| = 0.150004) DEBYE
SPIN SZ:         0.500000
SPIN S-SQUARED: 0.756731 (exact: 0.75)

Computing ESP on molecular grid...
Using layered Connolly surfaces for molecular grid...Total number of ESP grid points from 4 layers: 293
Electrostatic potential written to file ./scr.wat2/esp.xyz (unit: Ansgtrom)
Fitting RESP charges to the computed ESP...
ESP unrestrained charges:
Atom          X          Y          Z     Charge   Exposure
-----------------------------------------------------------
   O  -1.152733   2.016338   0.869274  -0.280321     1.0894
   H  -0.597153   0.332592   0.667073   0.637529     1.3065
   H  -1.738548   3.713312   1.190527   0.642792     1.2581
-----------------------------------------------------------
Quality of fit (RRMS):0.0196

Solving Resp charge with restraints iteratively...
Restraint parameters a:0.0005 , b:0.001 
ESP restrained charges:
Atom          X          Y          Z     Charge   Exposure
-----------------------------------------------------------
   O  -1.152733   2.016338   0.869274  -0.270220     1.0894
   H  -0.597153   0.332592   0.667073   0.632397     1.3065
   H  -1.738548   3.713312   1.190527   0.637824     1.2581
-----------------------------------------------------------
Quality of fit (RRMS):0.0195


Running Mulliken population analysis...


Gradient units are Hartree/Bohr
---------------------------------------------------
        dE/dX            dE/dY            dE/dZ
    0.0069999056    -0.0215342490    -0.0021086542
   -0.0127026554     0.0382694678     0.0049376063
    0.0057027498    -0.0167352188    -0.0028289520
---------------------------------------------------
Net gradient:  2.620300e-15 -3.504141e-15 -1.186985e-15
Net torque:    2.669500e-08  3.818745e-08  1.158203e-08

Writing out molden info
| Total processing time: 2.28 sec 

| Job finished: Wed Feb 19 10:44:32 2025

